code/parachain/frame/assets-registry/src/runtime.rs

```diff
@@ -69,6 +69,7 @@ ord_parameter_types! {
 
 parameter_types! {
 	pub const NativeED: Balance = 0;
+	pub const PicassoNetworkId: [u8; 4] = [1,0,0,0];
```

Comment:
can we be sure that newtwork id is just  nunber. pub xonst PNI : u32 = 1u32?

Reply:
so when takes asset id, and cats 32 prefix bits, and cast bits into u32 from array he gets 1?

Reply:
thisnis tesr. butl just asking for future

Reply:
Can it not be number? We can interpret any 32 bits as u32. Do you mean you want to specify it as a number in a config for Picasso?

Reply:
I will double check that given asset id, we retrieve correct network id

Reply:
i want to type one 1 in pd.js and in cosmos cli, instead 0x something. if it will be in config const 1 would be great too as that value can be seen in consts of pallet in pd.js

Reply:
changed to u32 instead of [u8;4]